ambassador/tests/t_extauth.py

```diff
@@ -174,6 +174,13 @@ def config(self):
 apiVersion: ambassador/v0
 kind:  Module
 name:  ambassador
+config:
```

Comment:
This doesn't seem like it can work -- only one `ambassador` `Module` is allowed? 

This test should probably `check` that there are no errors in loading the config.